### PR TITLE
chore: remove HPC specific code

### PR DESCRIPTION
**Pull request by nguyen102**
Issue #, if available:

Description of changes:
chore: remove HPC specific code

**Testing**
Was able to start from this branch and run `updatePlugins.sh` to get REACT App starter app in SWB

Checklist:

* [ ] Have you successfully deployed to an AWS account with your changes?
* [ ] Have you written new tests for your core changes, as applicable?

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: solutions/swb-reference/src/backendAPI.ts [hunk 1]
```diff
@@ -31,38 +30,7 @@ const aws: AwsService = new AwsService({
...
-      service: new HPCService()
-    },
-    {
-      path: '/projects/:projectId/clusters/:clusterName/headNode/:instanceId/jobs/:jobId/cancel',
-      serviceAction: 'cancelJob',
-      httpMethod: 'put',
-      service: new HPCService()
-    }
-  ],
+  routes: [],
```
Comment (rangfeli) [inline solutions/swb-reference/src/backendAPI.ts:33]: similar to the develop branch we should add back:
routes: [
    {
      path: '/foo',
      serviceAction: 'launch',
      httpMethod: 'post',
      service: new SagemakerNotebookEnvironmentLifecycleService()
    }
  ]

File: solutions/swb-ui/scripts/apps_menu.js [hunk 2]
```diff
@@ -3,9 +3,15 @@ const path = require('path');
...
-    })
+    fs
+      .readdirSync(pagePath)
+      .map((file) => {
+        if (path.extname(file).toLowerCase() === '.tsx') {
+          return path.parse(file).name;
+        }
+      })
+      .filter((file) => {
+        // Filer out `null` files
```
Comment (rangfeli) [inline solutions/swb-ui/scripts/apps_menu.js:14]: Simple to correction to "filter"

File: solutions/swb-ui/scripts/updatePlugins.sh [hunk 3]
```diff
@@ -11,7 +11,7 @@ for plugin in *; do #Iterate through swb-plugins
         pluginUpper=$(echo "${plugin}" | tr “[a-z]” “[A-Z]”);
         if ! $(jq --arg pluginName $pluginPackageName 'any(.projects[].packageName == $pluginName; .)' ../../rush.json); then
             echo "Adding $pluginPackageName to rush.json";
-            jq --arg pluginName $pluginPackageName --arg pluginFolder "solutions/swb-plugins/$plugin" '.projects += [{"packageName": $pluginName, "projectFolder": $pluginFolder, "reviewCategory": "production", "shouldPublish": true}]' ../../rush.json > ../../output_rush.json;
+            jq --arg pluginName $pluginPackageName --arg pluginFolder "solutions/swb-plugins/$plugin" '.projects += [{"packageName": $pluginName, "projectFolder": $pluginFolder, "reviewCategory": "production", "shouldPublish": false}]' ../../rush.json > ../../output_rush.json;
```
Comment (rangfeli) [inline solutions/swb-ui/scripts/updatePlugins.sh:14]: Why set shouldPublish equal to `false`?

File: solutions/swb-reference/src/backendAPI.ts (hunk 1 above)
Comment (nguyen102) [reply solutions/swb-reference/src/backendAPI.ts:33]: updated

File: solutions/swb-ui/scripts/updatePlugins.sh (hunk 3 above)
Comment (nguyen102) [reply solutions/swb-ui/scripts/updatePlugins.sh:14]: setting it to true will set our pipeline to publish the package as an `npm` package to NPM

File: solutions/swb-ui/scripts/updatePlugins.sh [hunk 4]
```diff
@@ -56,5 +56,5 @@ for page in ../swb-ui/src/pages/apps/*; do #Iterate through all items in pages/a
 done 
 echo "Running rush update";
 rush update;
-echo "Fininshed Updating of Plugins";
+echo "Finished Updating of Plugins";
```
Comment (rangfeli) [inline solutions/swb-ui/scripts/updatePlugins.sh:59]: Also make sure that package.json of swb-ui has Create_React_App removed

Comment (nguyen102) [reply solutions/swb-ui/scripts/updatePlugins.sh:59]: I'm sorry, what's the action item for the comment above? Is it additional text I should add to the message?

File: solutions/swb-ui/scripts/updatePlugins.sh (hunk 3 above)
Comment (rangfeli) [reply solutions/swb-ui/scripts/updatePlugins.sh:14]: understood, then we'll keep it to `false`, however for it to build everything we'll need to tell users to set it to `true`.

File: solutions/swb-ui/scripts/updatePlugins.sh (hunk 4 above)
Comment (rangfeli) [reply solutions/swb-ui/scripts/updatePlugins.sh:59]: Sorry ignore the item. I thought that the package.json of swb-ui wasn't updated with the removed dependency, but it's correct.

File: solutions/swb-ui/scripts/updatePlugins.sh (hunk 3 above)
Comment (nguyen102) [reply solutions/swb-ui/scripts/updatePlugins.sh:14]: hmm, I was able to test it from this branch with the value set to `false` and everything work as expected. Can you test on your side to see if everything build correctly for you as well?

File: solutions/swb-ui/scripts/apps_menu.js (hunk 2 above)
Comment (nguyen102) [reply solutions/swb-ui/scripts/apps_menu.js:14]: updated

File: solutions/swb-ui/scripts/updatePlugins.sh (hunk 3 above)
Comment (rangfeli) [reply solutions/swb-ui/scripts/updatePlugins.sh:14]: It build correctly with it set to `false` so we'll keep it like that. No need to set to `true`.

**Review by rangfeli — APPROVED** [review]
All ok!

**Pull request merged**